libs/input-mapping/src/Configuration/Configuration.php

```diff
@@ -1,19 +1,19 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Keboola\InputMapping\Configuration;
 
 use Symfony\Component\Config\Definition\ConfigurationInterface;
 use Symfony\Component\Config\Definition\Processor;
 
 abstract class Configuration implements ConfigurationInterface
 {
-    /**
-     * Shortcut method for processing configurations
-     *
-     * @param $configurations
-     * @return array
-     */
-    public function parse($configurations)
+    final public function __construct()
```

Comment:
Kdyby tu nebyl ten `final`, vubec bych si toho nevsiml. Nekde v kodu je predpokladam neco nasledjiciho?

```
/** @var class-string<Configuration> $configType */
new $configType();
```

Reply:
je to hnedle pod tím https://github.com/keboola/platform-libraries/blob/c330c9865dfd410d25c39c0761e92401b22d3e71/libs/input-mapping/src/Configuration/Configuration.php#L19 a phpstanu se to nelibí https://phpstan.org/blog/solving-phpstan-error-unsafe-usage-of-new-static ten final tu celkem sedí

Reply:
:facepalm: jasne